src/bun.js/test/jest.zig

```diff
@@ -1162,6 +1190,7 @@ pub const TestScope = struct {
     promise: ?*JSInternalPromise = null,
     ran: bool = false,
     task: ?*TestRunnerTask = null,
+    snapshotCount: u32,
```

Comment:
this should be underscore_case to match zig convention. We differ from that convention for globalObject/globalThis in some cases but that's usually it